backend/src/routers/gameRouter.ts

```diff
@@ -242,32 +269,64 @@ gameRouter.post('/remove', async (req, res) => {
 	try {
 		await removeGame(req.body.id);
 		res.status(200).json({ message: 'Game removed' });
+	} catch (e) {
+		if (e instanceof Error) res.status(400).json({ message: e.message });
+		else res.status(400).json({ message: 'Error removing game' });
 	}
-	catch (e) {
-		if (e instanceof Error)
-			res.status(400).json({ message: e.message });
-		else
-			res.status(400).json({ message: 'Error removing game' });
-	}
-})
+});
+
+/**
+ * @api {get} /api/v1/games/owners Get all game owners
+ * @apiName GetOwners
+ * @apiGroup Games
+ * @apiDescription Gets all game owners
+ *
+ * @apiSuccess {Object[]} Owners List of game owners
+ *
+ * @apiSuccessExample Success-Response:
+ * HTTP/1.1 200 OK
+ * [
+ *  {
+ *   "id": "clgkri8kk0000przwvkvbyj95",
+ *   "name": "Game Owner 1"
+ *  },
+ *  {
+ *   "id": "clgkri8ku0000przwvkvbyj95",
+ *   "name": "Game Owner 2"
+ *  }
+ * ]
+ **/
+gameRouter.get('/owners', async (req, res) => {
+	const owners = await getGameOwnersWithGames();
 
+	const formattedOwners = await Promise.all(
+		owners.map(async (owner) => ({
+			id: owner.id,
+			name: await getGameOwnerNameFromId(owner.id)
+		}))
```

Comment:
Won't this crash as soon as we add a single organization game owner as `getGameOwnerNameFromId` expects its input to be a user game owner?

Reply:
I just thought we would update the function to retrieve the organization name at that point... Otherwise I don't really see a way to fix this now?

Reply:
Ok, as we currently don't have an endpoint to add orgs as game owners I can accept it. We just need to remember that this needs to be updated as soon as we add orgs as game owners.

Reply:
Actually, I'm derp. Forgot that we actually have a name field in the database. Updating now...

Reply:
Fixed now